src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,114 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User defaultUser;
+    private static User anotherUser;
+    private static RegistrationService service;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+        defaultUser = new User();
+        anotherUser = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        defaultUser.setLogin("Mate");
+        defaultUser.setPassword("more_than_six");
+        defaultUser.setAge(25);
+        anotherUser.setLogin("Academic");
+        anotherUser.setPassword("six_or_more");
+        anotherUser.setAge(50);
```

Comment:
And also, you can add a constructor to the User class and initialize with declaration , like
 anotherUser = new User("Mate", "more_than_six", 25);

Reply:
Oh, I thought we can modify only one class